fixed_obstacles : {
"Iceberg_1" : "RADIUS=15 COORD=10,470952/59,436573",
"Iceberg_2" : "RADIUS=15 COORD=10,469416/59,437963",
"Iceberg_3" : "RADIUS=15 COORD=10,468931/59,439197",
"Iceberg_4" : "RADIUS=15 COORD=10,474656/59,439714",
"Iceberg_5" : "RADIUS=15 COORD=10,477303/59,439172",
"Iceberg_6" : "RADIUS=15 COORD=10,474814/59,437146",
"Iceberg_7" : "RADIUS=15 COORD=10,480263/59,441991",
"Iceberg_8" : "RADIUS=15 COORD=10,474666/59,443562",
"Iceberg_9" : "RADIUS=15 COORD=10,467777/59,441583",
"Iceberg_10" : "RADIUS=15 COORD=10,483189/59,436865",
"Iceberg_11" : "RADIUS=15 COORD=10,481345/59,434604",
"Iceberg_12" : "RADIUS=15 COORD=10,465075/59,433254",
"Iceberg_13" : "RADIUS=15 COORD=10,472437/59,437849",
"Iceberg_14" : "RADIUS=15 COORD=10,472209/59,438807"
}

ships : {
"support_vessel" : "LENGTH=30 SpeedInKnots=1 WP=10,468943/59,440308 WP=10,474462/59,440736 WP=10,480279/59,439919 WP=10,481580/59,437513",
"research_ship" : "LENGTH=100 SpeedInKnots=10 WP=10,483932/59,438875 WP=10,460502/59,432853"
}
